### PR TITLE
Edit descriptions of `influxd` command

### PR DESCRIPTION
**Pull request by pierwill**
`influxd` is the command to run the db, not a full CLI.

Also renames files and fixes links.

### CODE REVIEW

**Comment by pierwill** [comment]
Closes #2821.

File: content/influxdb/v1.8/tools/_index.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ This section covers the available tools for interacting with InfluxDB.
 
 The InfluxDB command line interface (`influx`) includes commands to manage many aspects of InfluxDB, including databases, organizations, users, and tasks.
 
-## [`influxd` command line interface (CLI)](/influxdb/v1.8/tools/influxd-cli/)
+## [`influxd` command](/influxdb/v1.8/tools/influxd)
```
Comment (sanderson) [inline content/influxdb/v1.8/tools/_index.md:19]: ~command~ -> daemon

File: content/influxdb/v1.8/tools/_index.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ This section covers the available tools for interacting with InfluxDB.
...
 
-The `influxd` command line interface (CLI) starts and runs all the processes necessary for InfluxDB to function.
+The `influxd` command starts and runs all the processes necessary for InfluxDB to function.
```
Comment (sanderson) [inline content/influxdb/v1.8/tools/_index.md:21]: ~command~ -> daemon

File: content/influxdb/v1.8/tools/_index.md (hunk 1 above)
Comment (timhallinflux) [reply content/influxdb/v1.8/tools/_index.md:19]: +1

File: content/influxdb/v1.8/tools/_index.md (hunk 2 above)
Comment (timhallinflux) [reply content/influxdb/v1.8/tools/_index.md:21]: +1

**Pull request merged**